lua/customroles/elementalist/sv_elementalist.lua

```diff
@@ -138,6 +138,10 @@ hook.Add("EntityTakeDamage", "Elementalist Effects", function(ent, dmginfo)
         if att:HasEquipmentItem(EQUIP_ELEMENTALIST_WINDBURN_UP) and GetChanceConVarOutcome("ttt_elementalist_windburn+_launch_chance") then
             --Upgrade functionality
             ent:SetVelocity(ent:GetVelocity() + Vector(0, 0, 1000 + (1000 * scale)))
+
+            if ply.SetJumpLevel and ply.GetMaxJumpLevel then
```

Comment:
Did you test this? I'm wondering if there needs to be a slight delay so the player is actually in the air

Reply:
> Can't really test without another person, figured there'd be some general playtesting done before releasing the pack and I could test this and other functionality then.

If you do `bot_mimic 1` they will jump when you jump so you can test that way

Reply:
MIMIC bot? I had no idea such a thing existed.

There's a small handful of other things that will still need to be tested regardless if I can prove the double jump removal works or not. Is it okay to wait or do you adamantly want everything that can be tested/working tested and working before any playtesting?

Reply:
We don't normally do a formal test of things, it's more like we release it to our friend group and try it out there. 

We'd like to make sure the basic functionality works before then so we're not spending time fixing errors while they just want to play. 

The timeline is very relaxed here so it doesn't need to be done right away. Also I'm probably going to test things out myself just so I'm familiar with the roles anyway so it's not a big deal if you can't test right now.

Reply:
Yea I'm not anticipating anything BREAKS, only that I haven't seen how certain effects will work when playing, if certain inputs will actually register, how screen overlays look, etc. If you're content waiting, feel free to resolve the conversation. I went ahead and added a tick wait anyway.